src/interfaces/stepper-interfaces/DefineCompany/define-company-interface.ts

```diff
@@ -1,6 +1,7 @@
-import { BusinessContactIF } from './business-contact-interface'
+import { BusinessContactIF, NameRequestIF } from '@/interfaces'
 
 export interface DefineCompanyIF {
-    valid: boolean
+    valid: boolean;
     businessContact : BusinessContactIF;
+    nameRequest: NameRequestIF;
```

Comment:
I consider NR data as a separate block of data, separate from whatever the Define Company step does, so I'd prefer to see this at a "top level" instead of nested here. Thoughts?

Reply:
Good point for sure, it certainly feels a little more universal beyond step one as the data won't be determined by step one but just display during it. I'm good with that change!